src/txn/txn_recover.c

```diff
@@ -633,6 +622,51 @@ __wt_txn_recover(WT_SESSION_IMPL *session)
         WT_ERR(ret);
     }
 
+    /*
+     * We should check whether the history store file exists in the metadata or not. or not. If it
+     * does not, then we should not apply rollback to stable to each table. This might happen if
```

Comment:
I suggest avoiding the double negative and say, `If it does not exist, then we should skip rollback to...`